### PR TITLE
Document tagging examples in the Release Guide

### PR DESCRIPTION
**Pull request by ppalaga**

### CODE REVIEW

**Review by oscerd — APPROVED** [review]

File: docs/modules/ROOT/pages/contributor-guide/release-guide.adoc [hunk 1]
```diff
@@ -233,13 +233,61 @@ Check the full [release announcement](https://camel.apache.org/blog/2021/06/came
...
+* Make sure all https://github.com/apache/camel-quarkus-examples/pulls[PRs] against `camel-quarkus-main` branch are merged.
+* Upgrade to the Camel Quarkus version you just released
++
+[source,shell]
+----
+$ NEW_CQ_VERSION=... # E.g. 2.2.0
+$ git fetch upstream
+$ git checkout camel-quarkus-main
+$ git reset --hard upstream/camel-quarkus-main
+$ find . -type f -name pom.xml -exec sed -i "s|<camel-quarkus.version>[^<]*</camel-quarkus.version>|<camel-quarkus.version>$NEW_CQ_VERSION</camel-quarkus.version>|g" {} \;
```
Comment (aldettinger) [inline docs/modules/ROOT/pages/contributor-guide/release-guide.adoc:247]: Can't we do it with `./mvnw-for-each.sh versions:set` ?

File: docs/modules/ROOT/pages/contributor-guide/release-guide.adoc [hunk 2]
```diff
@@ -233,13 +233,61 @@ Check the full [release announcement](https://camel.apache.org/blog/2021/06/came
...
++
+* If everything works for you locally, open a PR to merge `camel-quarkus-main` to `main`
+* Once the PR is merged, tag the `main` branch with the `$NEW_CQ_VERSION`:
++
+[source,shell]
+----
+$ git checkout main
+$ git fetch upstream
+$ git reset --hard upstream/main
+$ ./mvnw-for-each.sh versions:set -DnewVersion=$NEW_CQ_VERSION
```
Comment (aldettinger) [inline docs/modules/ROOT/pages/contributor-guide/release-guide.adoc:267]: I don't catch something. Are we setting the versions a second time ? Feel like it was already done on camel-quarkus-main and then merged ?

File: docs/modules/ROOT/pages/contributor-guide/release-guide.adoc (hunk 1 above)
Comment (ppalaga) [reply docs/modules/ROOT/pages/contributor-guide/release-guide.adoc:247]: AFAIK `versions:set` can only set project version and cannot set a property version. But maybe there is another mojo able to that.

File: docs/modules/ROOT/pages/contributor-guide/release-guide.adoc (hunk 2 above)
Comment (ppalaga) [reply docs/modules/ROOT/pages/contributor-guide/release-guide.adoc:267]: Yes, here we set the `project.version` s of the individual examples. Above, we have set the value of `camel-quarkus.version` property.

File: docs/modules/ROOT/pages/contributor-guide/release-guide.adoc (hunk 1 above)
Comment (aldettinger) [reply docs/modules/ROOT/pages/contributor-guide/release-guide.adoc:247]: ok, this was part of the misunderstanding. so, that's fine.

File: docs/modules/ROOT/pages/contributor-guide/release-guide.adoc (hunk 2 above)
Comment (aldettinger) [reply docs/modules/ROOT/pages/contributor-guide/release-guide.adoc:267]: Indeed, a good sleep after, it's clearer now :)

**Review by aldettinger — APPROVED** [review]
Looks good to me and tagging is a good idea :+1:

**Comment by ppalaga** [comment]
Thanks for the review, @aldettinger !

**Pull request merged**